.github/workflows/testscorev12.yml

```diff
@@ -0,0 +1,51 @@
+name: Test service_bw2 for TYPO3 12 LTS
+
+on: pull_request
+
+jobs:
+  CGL:
+    name: Coding Style Check (TYPO3 Community CGL)
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Composer
+        run: Build/Scripts/runTests.sh -t 12 -p 8.1 -s composerUpdate
+
+      - name: Lint PHP
+        run: Build/Scripts/runTests.sh -t 12 -p 8.1 -s lint
+
+      - name: Validate code against CGL
+        run: Build/Scripts/runTests.sh -t 12 -p 8.1 -s cgl -n
+
+  testsuite:
+    name: PHP Unit and Functional Tests for TYPO3 Version 12 LTS
+
+    needs: CGL
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        php:
+          - '8.1'
+          - '8.2'
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Composer
+        run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -t 12 -s composerUpdate
+
+      - name: Unit Tests
+        run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -s unit
+
+      - name: Functional tests with mariadb
+        run: Build/Scripts/runTests.sh -p ${{ matrix.php }} -d mariadb -s functional
```

Comment:
No Problems. :) I did'nt even noticed that it was'nt there from my copied configuration. Because it failed in glossary2 and I copied the test suite file from there :)